test/javascript/components/concept-map/ConceptMap.test.tsx

```diff
@@ -1,105 +1,94 @@
 // Deps
 import React from 'react'
+import '@testing-library/jest-dom'
```

Comment:
It's fine to have this in every test file, but you probably want it in a [`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array).